From d73851f95836a057fffbaf5fd49f25bdb9a0ec35 Mon Sep 17 00:00:00 2001
From: fosero <fosero@users.noreply.github.com>
Date: Thu, 1 Sep 2016 10:09:14 +0200
Subject: [PATCH 1/1] meson.build: Fix typo

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index cb49019..bf22230 100644
--- a/meson.build
+++ b/meson.build
@@ -18,7 +18,7 @@ if get_option('do-post-install')
 endif

 if WITH_PLAYER_GSTREAMER_CLUTTER
-   subprojec('player-backend-gstreamer-clutter')
+   subproject('player-backend-gstreamer-clutter')
 endif
 if WITH_PLAYER_MPV_OPENGL
    subproject('player-backend-mpv-opengl')
--
2.9.3

